README.md

```diff
@@ -35,6 +37,16 @@ At present the cache does not expire automatically - you should delete the files
 Currently, we only support the [OpenStreetMap](http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames) convention for tile URLs.
 This e.g. implies that only raster tiles (no vector tiles) are supported.
 
+Transformation of tiles to RViz fixed frame can be done in two ways that are configured using `Map Transform Type` option:
+
+1. Specify a `Map frame`, which is an ENU-oriented frame in which your robot localizes. This mode expects that the frame
+   of the subscribed `NavSatFix` messages is consistent with the measured latitude/longitude in this map frame.
+   In this mode, the tiles go through an intermediate transform to the map frame.
+2. Specify `UTM frame` (and possibly `UTM zone`). In this mode, no map frame is required and the tiles are directly 
+   placed on their UTM positions. This mode expects UTM frame is represented in your transform tree. The subscribed
+   `NavSatFix` messages are only used to determine the tiles to download, so small inconsistencies between the
```

Comment:
Question: can't we drop the lat/lon completely in this case? As per discussion on 'fake gps data'.

Your proposal:
```yaml
Topic: gps/fix
Map Transform Type: UTM frame
UTM Frame: utm
```

But why not do everything in TF and drop the requirement of GPS completely

```yaml
Map Transform Type: UTM frame
UTM Frame: utm
Base Link Frame: base_link
```

Reply:
This is exactly what I wanted. #106 is the direction towards this goal.